epub33/core/index.html

```diff
@@ -2773,12 +2792,12 @@ <h6>The <code>item</code> Element</h6>
 							</dl>
 
 							<p>Each <code>item</code> element identifies a <a>Publication Resource</a> by the URL
-								[[URL]] in its <code>href</code> attribute. EPUB Creators MAY use <a
-									href="https://url.spec.whatwg.org/#absolute-url-string">absolute-</a> or <a
-									href="https://url.spec.whatwg.org/#relative-url-string">relative-URL string</a>
-								[[URL]], but they MUST ensure each URL is unique within the <code>manifest</code> scope
-								after <a href="https://www.w3.org/TR/epub-rs-33/#sec-pkg-doc-relative-urls">resolution
-									to an absolute URL</a> [[EPUB-RS-33]].</p>
+								[[URL]] in its <code>href</code> attribute. 
+
+								The value MUST be a an <a
+									href="https://url.spec.whatwg.org/#absolute-url-string">absolute-</a> or  <a href="https://url.spec.whatwg.org/#path-relative-scheme-less-URL">path-relative-scheme-less-URL</a> string. EPUB Creators MUST ensure each URL is unique within the <code>manifest</code> scope
+									after <a href="#parsing-urls-in-the-package-document">resolution
+									to an absolute URL</a>.</p>
```

Comment:
nit: s/resolution to an absolute URL/parsing/

Reply:
yep.